src/12-standard-library.md

```diff
@@ -0,0 +1,11 @@
+# Standard library
```

Comment:
The `l` in `library` should be capital, I think.

Reply:
You're right, I'll update it

Reply:
Thank you :)